oap-server/server-query-plugin/query-graphql-plugin/src/main/java/org/apache/skywalking/oap/query/graphql/resolver/LogQuery.java

```diff
@@ -62,20 +65,26 @@ public Logs queryLogs(LogQueryCondition condition) throws IOException {
         }
         Order queryOrder = isNull(condition.getQueryOrder()) ? Order.DES : condition.getQueryOrder();
         condition.getTags().forEach(tag -> {
-            tag.setKey(tag.getKey().trim());
-            tag.setValue(tag.getValue().trim());
+            if (tag != null) {
```

Comment:
You should check the error logs, I am not sure it is tags or tag is null. But I prefer tags.